sdk/src/jumbf_io.rs

```diff
@@ -33,6 +33,8 @@ use crate::{
 lazy_static! {
     static ref ASSET_HANDLERS: HashMap<String, Box<dyn AssetIO>> = {
         let handlers: Vec<Box<dyn AssetIO>> = vec![
+            #[cfg(feature = "pdf")]
+            Box::new(crate::asset_handlers::pdf_io::PdfIO::new("")),
```

Comment:
Probably better to put the use statement up top behind the feature, since you refer to it more than once